app/src/androidTest/assets/inserts.txt

```diff
@@ -82,3 +88,17 @@ INSERT INTO "talks" VALUES(46,'Opening Talk','',6,96,'/teacher/96/talk/46/200602
 INSERT INTO "talks" VALUES(47,'Wisdom & Compassion','',6,96,'/teacher/96/talk/47/20060208-Joseph_Goldstein-IMSRC-wisdom_compassion-47.mp3',59.366667,'2011-07-24 22:28:52','2006-02-08 00:00:00',3813,'');
 INSERT INTO "talk_stars" VALUES(46);
 INSERT INTO "centers" VALUES(1,'','','Unknown');
+
```

Comment:
I didn't know about the `.mode insert` feature. Good to know! Since we already have the basic DB structure in `insersts.txt`, this makes it quite easy to extend the test DB in future even without a dedicated script.